### PR TITLE
Update IAB Workshop.md

### PR DESCRIPTION
**Pull request by bclaise**

### CODE REVIEW

File: IAB Workshop.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
 # Introduction
-The IAB organized a workshop in 2002 to establish a
-dialog between network operators and protocol developers, and to guide the IETF focus on work regarding network management. That workshop identified 14 requirements that the operators wanted to see in the next generation of management, and was documented in the "IAB Network Management Workshop" (RFC 3535). The RFC was instrumental in the development of NETCONF, YANG, and RESTCONF.
+The IAB has organized a workshop (June 4-June 6, 2002) to establish a dialog between network operators and protocol developers, and to
```
Comment (mjethanandani) [inline IAB Workshop.md:2]: s/IAB has organized/IAB had organized/

File: IAB Workshop.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+guide the IETF focus on work regarding network management. The outcome of that workshop was documented in the "IAB Network
+Management Workshop" (RFC3535) that identified 14 requirements that the operators wanted to see in the next generation of management
 
-This IAB workshop should look at whether those requirements been met, and more importantly gather input on what are some of the new requirements the operators are facing.
+Those requirements were instrumental for developing first the NETCONF protocol (in the NETCONF Working Group), then YANG data modeling language
```
Comment (mjethanandani) [inline IAB Workshop.md:6]: s/then YANG/and YANG/

File: IAB Workshop.md [hunk 3]
```diff
@@ -26,8 +31,13 @@ in public sources like Sonic or FRR?
...
 
+# What this workshop would not cover
+- encryption in operator environments
+- what else?
+
 # Outreach
 In 2002, there were enough operators attending IETF meetings. Those numbers have dwindled over time, with some of the operators attending conferences that are more operator focused. These include RIPE, NANOG, AutoConn etc.
 
-As such the IAB may have to do an outreach effort to get the input they are looking for. A tentative plan is to target these conferences in the October/November 2024 timeframe, followed by a virtual workshop in December to discuss the findings. 
+As such the IAB may have to do an outreach effort to get the input they are looking for. A tentative plan is to target these conferences in the October/November 2024 timeframe, and virtual workshops, followed by a final virtual calls tentatively in December to discuss the findings.
```
Comment (mjethanandani) [inline IAB Workshop.md:41]: Were we planning multiple virtual workshops after the conferences, and multiple virtual calls in December? I thought Wes mentioned one.

**Review by mjethanandani — APPROVED** [review]
The document looks good to me, except for a couple of nits, and one question.

File: IAB Workshop.md (hunk 2 above)
Comment (hardaker) [reply IAB Workshop.md:6]: agree

File: IAB Workshop.md (hunk 3 above)
Comment (hardaker) [reply IAB Workshop.md:41]: I think that's up for debate and we should plan that in a larger meeting.  Anything is possible depending on what we want to commit to doing :-)

**Comment by hardaker** [comment]
FYI, I'm going to merge this so I can add my own changes afterward.

**Pull request merged**